<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-20DAHI5" name="org.winvinaya.academy.testauto.test.testsuites.WebTestUserLoadingTime" tests="2" failures="0" timestamp="08 Jun 2022 05:10:53 GMT" time="45.813" errors="1">
  <testcase name="afterSuite" time="1.895" classname="org.winvinaya.academy.testauto.test.testsuites.WebTestUserLoadingTime">
    <error type="java.lang.RuntimeException" message="javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials u1-20020a056a00098100b0051bc4ed56bcsm6196069pfg.204 - gsmtp
">
      <![CDATA[java.lang.RuntimeException: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials u1-20020a056a00098100b0051bc4ed56bcsm6196069pfg.204 - gsmtp

at org.winvinaya.academy.testauto.SendAttachment.sendmail(SendAttachment.java:102)
at org.winvinaya.academy.testauto.test.WinVinayaAcadamyWebBaseTest.afterSuite(WinVinayaAcadamyWebBaseTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials u1-20020a056a00098100b0051bc4ed56bcsm6196069pfg.204 - gsmtp

at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:965)
at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:876)
at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:780)
at javax.mail.Service.connect(Service.java:388)
at javax.mail.Service.connect(Service.java:246)
at javax.mail.Service.connect(Service.java:195)
at javax.mail.Transport.send0(Transport.java:254)
at javax.mail.Transport.send(Transport.java:124)
at org.winvinaya.academy.testauto.SendAttachment.sendmail(SendAttachment.java:96)
... 21 more
]]>
    </error>
  </testcase> <!-- afterSuite -->
  <testcase name="AcdemyLoginTest" time="43.918" classname="org.winvinaya.academy.testauto.test.testsuites.WebTestUserLoadingTime"/>
</testsuite> <!-- org.winvinaya.academy.testauto.test.testsuites.WebTestUserLoadingTime -->
